CHANGELOG.md

```diff
@@ -140,7 +140,7 @@ Documentation for rocFFT is available at
 
 ### Additions
 
-* Added gfx1101 to default `AMDGPU_TARGETS`
+* Added gfx1151 to default `AMDGPU_TARGETS`
```

Comment:
This entry needs to go into the section for 6.2.2 that @af-ayala is adding in #516.  So either we try to combine these two PRs, or the two of you need to coordinate to resolve your conflicts when these are approved.